config/settings.py

```diff
@@ -3,7 +3,7 @@
 
 class Settings(BaseSettings):
     database_url: str = "postgresql+asyncpg://authdb_owner:V97gEkaPchLf@ep-plain-water-a5scbycn.us-east-2.aws.neon.tech/authdb"
-    secret_key: str = "e705c2325058744e7328d842be8da79a1f7c934363b1f061c7934d4804f3d360"
+    secret_key: str = "empty_string_1"
```

Comment:
The database URL contains credentials, which is a security concern. It's recommended to use environment variables for database connections. This approach enhances security by not exposing credentials in the codebase and allows for easier configuration changes without code modifications.

**Label**: security

**Code suggestion**

```python
database_url: str = os.getenv("DATABASE_URL")
```